test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -583,7 +583,13 @@ foo
 
         It "-WorkingDirectory should be processed before profiles" {
 
-            $currentProfile = Get-Content $PROFILE
+            if (Test-Path $PROFILE) {
+                $currentProfile = Get-Content $PROFILE
+            }
+            else {
+                New-Item -ItemType File -Path $PROFILE -Force
```

Comment:
```suggestion
                $null = New-Item -ItemType File -Path $PROFILE -Force
```

Reply:
Merging as this comment is non-blocking.

Reply:
```suggestion
                New-Item -ItemType File -Path $PROFILE -Force > $null
```
😉